_studio/mfx_lib/decode/h264/src/mfx_h264_dec_decode.cpp

```diff
@@ -1597,11 +1592,7 @@ mfxStatus VideoDECODEH264::DecodeFrame(mfxFrameSurface1 *surface_out, UMC::H264D
     {
         index = m_FrameAllocator->FindSurface(surface_out, m_isOpaq);
         pFrame = m_pH264VideoDecoder->FindSurface((UMC::FrameMemID)index);
-        if (!pFrame)
-        {
-            VM_ASSERT(false);
-            return MFX_ERR_NOT_FOUND;
-        }
```

Comment:
it seems not be related with "VDBOX+SFC for 1->1 transcode scenarios". Could we add it into another commit?

Reply:
addressed